pyro/distributions/torch.py

```diff
@@ -39,6 +39,19 @@ def expand(self, batch_shape):
 
 
 class Categorical(torch.distributions.Categorical, TorchDistributionMixin):
+
+    # workaround for jit errors in broadcast_all.
+    # this can be deleted after https://github.com/pytorch/pytorch/pull/10321
+    if hasattr(torch, 'broadcast_tensors'):
```

Comment:
Would you mind either
1. cherry-picking 2a93a9a4e049 so that `OneHotCategorical` also gets a correct `log_prob`, or
2. reverting this change, since it's only available post-0.4.1?